operations/app/terraform/vars/skylight/secrets.tf

```diff
@@ -0,0 +1,7 @@
+## Let's get our secrets from the secrets key vault
+## Note, this will need to be pre-populated
+
+# data "azurerm_key_vault" "tf-secrets" {
+#  name                = var.tf_secrets_vault
+#  resource_group_name = var.resource_group
+# }
```

Comment:
Should this be commented out? (It very well might be, I just have 1% of an idea how terraform configurations work)

Reply:
good catch, I meant to remove this and forgot, will remove!